live-build/hooks/300-create-classic-diff.binary

```diff
@@ -12,7 +12,7 @@ PREFIX=binary/boot/filesystem.dir
 (cd $PREFIX
  mkdir -p var/lib/classic
  apt-get clean
- apt-get -y install --reinstall -d apt locales base-files libapt-inst2.0 libapt-pkg5.0 ubuntu-keyring
+ apt-get -y install --reinstall -d apt locales base-files libapt-inst2.0 libapt-pkg5.0 ubuntu-keyring lsb-release
```

Comment:
this code adds lsb-release to the classic snap tarball, it does not remove anything ...i think you wanted to comment on the code that removes the leftover dpkg cruft for lsb-release ... note that lsb_release itself has never been supported in core and was removed in the same code path before already, the new code just does this cleaner (by using dpkg to also remove the package leftovers)